transition-scripts/values/eb75ad/prod/values-gold-eb75ad-prod-active.yaml

```diff
@@ -0,0 +1,52 @@
+replicaCount: 3
+
+additionalServerOptions: >-
+  -Dkeycloak.profile.feature.impersonation=disabled
+  -Djboss.persistent.log.dir=/var/log/eap
+  -Djgroups.dns.query=sso-keycloak-ping.eb75ad-prod.svc.cluster.local
+
+extraEnvs:
+  - name: KC_HOSTNAME_URL
+    value: https://loginproxy.gov.bc.ca/auth/
+  - name: KC_HOSTNAME_ADMIN_URL
+    value: https://loginproxy.gov.bc.ca/auth/
+  - name: QUARKUS_LOG_FILE_ROTATION_MAX_BACKUP_INDEX
+    value: "21"
```

Comment:
I think this setting makes sense, but maybe a little high. Comes to about 19 gigs unzipped, usually compresses to less than half the size so would be inside our pvc size. But just want to flag it since we really don't need long term storage anymore we could probably reduce as well